examples/scripts/ExampleMultipleChaperones.py

```diff
@@ -22,7 +22,7 @@
 # Load beam
 beam = ps.Beam(os.path.join(pwd,'../input/beam/amo86615.beam'))
 
-geom = os.path.join(pwd,'../input/lcls/amo86615/PNCCD::CalibV1/Camp.0:pnCCD.1/geometry/0-end.data')
```

Comment:
this line is correct

Reply:
didn't work with this line

Reply:
geom = os.path.join(pwd,'../lcls/amo86615/PNCCD::CalibV1/Camp.0:pnCCD.1/geometry/0-end.data')

Reply:
I cleaned up the input folder. you will need to do this:
cd pysingfel/examples/input  
Download a copy of lcls.tar.gz in the /input folder from https://stanford.box.com/s/e7c30tvhfz0485j2xr48rnrvel8a4yby
tar -xvf lcls.tar.gz

Please update README if this doesn't work